packages/angular/src/table/table-model.class.ts

```diff
@@ -98,7 +116,7 @@ export class AITableModel implements PaginationModel {
   /**
    * Contains information about the header cells of the table.
    */
-  protected header: TableHeaderItem[][] = [[]];
+  protected header: AITableHeaderItem[][] = [[]];
```

Comment:
Hold on, this looks a bit breaky... Could we keep it as is?

If they don't need alignment, they just use the cca default.

Reply:
It would result in a type error when you check for alignment in the template if it was left as is. Can we maybe do `TableHeaderItem[][] | AITableHeaderItem[][]` and do some kind of type checking in the template?

Reply:
Perhaps... It wouldn't be a breaking change in that case :)